pkgs/top-level/all-packages.nix

```diff
@@ -30042,7 +30042,9 @@ with pkgs;
 
   ladspa-sdk = callPackage ../applications/audio/ladspa-sdk { };
 
-  ladybird = qt6.callPackage ../applications/networking/browsers/ladybird { };
+  ladybird = qt6Packages.callPackage ../applications/networking/browsers/ladybird {
+    stdenv = if stdenv.isDarwin then llvmPackages_14.stdenv else gcc11Stdenv;
+  };
```

Comment:
Why is this in all-packages instead of ladybird? Tbh I have yet to understand what is the accepted convention myself...

Reply:
I'm doing this since there were already a lot of stdenv overrides in all-packages.nix. But it doesn't really matter, I can move that to ladybird if you want.

Reply:
@fgaz this makes it easier for others to use `ladybird.override` to change what it uses for `stdenv`.

Reply:
Thanks for the clarification @ElvishJerricco. Does this apply to all package variants or just stdenv?